docs/data/base/components/input/input.md

```diff
@@ -0,0 +1,41 @@
+---
+product: base
+title: React Input unstyled component and hook
+components: InputUnstyled
+githubLabel: 'component: input'
+packageName: '@mui/base'
+---
+
+# Input
+
+<p class="description">Inputs let users enter and edit text.</p>
+
+## Basic usage
+
+```js
+import InputUnstyled from '@mui/base/InputUnstyled';
+```
+
+{{"demo": "UnstyledInputBasic.js", "defaultCodeOpen": false}}
+
+## Adornments
+
+Sometimes it is useful to add some kind of prefix, a suffix, or an action to an input.
+You can achieve this by using the adornment props.
+For instance, you can use an icon button to hide or reveal the password.
+
+{{"demo": "InputAdornments.js", "defaultCodeOpen": false}}
```

Comment:
Should we have the `InputAdornments` component return code snippet visible? It seems useful to showcase the `adornment` prop in action within the component.

Reply:
We decided for all unstyled demos to disable the preview until https://github.com/mui/material-ui/issues/31075 is implemented.